src/main/java/com/hibernate/cinema/controller/UserController.java

```diff
@@ -22,7 +22,7 @@ public UserController(UserService userService, UserMapper userMapper) {
     }
 
     @GetMapping("/by-email")
-    public UserResponseDto getUserByEmail(@RequestParam String email) {
```

Comment:
this is not supposed to be changed

Reply:
`Replace all ?userId @RequestParam in your code with Authentication object` checklist says. I used email instead of id

Reply:
the method is called `getUserByEmail()`)